service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseUrlService.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.CASE_TYPE;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.JURISDICTION;
+
+@Service
```

Comment:
minor: currently only one dependency but we can use `@RequiredArgsConstructor(onConstructor = @__({@Autowired}))`.

Reply:
done